sample/Makefile

```diff
@@ -1,4 +1,4 @@
-all:
+all: hl
```

Comment:
Does it make sense that "all" include both HL and JS?

Reply:
No. I was wrong. It should be `all: hl js`, right?

Reply:
Yep.